google/cloud/storage/client_object_copy_test.cc

```diff
@@ -63,7 +63,7 @@ TEST_F(ObjectCopyTest, CopyObject) {
   std::string text = R"""({
       "name": "test-bucket-name/test-object-name/1"
 })""";
```

Comment:
Is it important to not indent here (and line 133)?

Reply:
No, these are JSON strings (embedded in C++ raw string literals), spaces are ignored. I have not convinced myself of which style is better for these. Well, I really like using the raw string literals, because JSON is full of double quotes (and braces!). But their internal indentation, and their indentation within the program I am not sure what is best, suggestions are welcome.

Reply:
If spaces are ignored, I think I prefer the indented version (clang-format seems to leave them alone).  But no big deal either way.  Consistency should probably be the only goal, and at the moment it looks like we have 60 occurrences without indentation and 45 with.